apps/console/src/features/userstores/components/edit/edit-basic-details-user-store.tsx

```diff
@@ -307,31 +321,43 @@ export const EditBasicDetailsUserStore: FunctionComponent<EditBasicDetailsUserSt
             operation: "REPLACE",
             path: `/properties/${ name }`,
             value: data.checked ? "false" : "true"
-        }
+        };
 
-        patchUserStore(id, [ patchData ]).then(() => {
-            setEnabled(data.checked);
-
-            dispatch(addAlert({
-                description: t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.success.description"),
-                level: AlertLevels.SUCCESS,
-                message: t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.success.message")
-            }));
-            // Prevent update until the userstore update delay is fixed.
-            //update();
-        }).catch(error => {
-            dispatch(addAlert({
-                description: error?.description
-                    || t("adminPortal:components.userstores.notifications." +
-                        "updateUserstore.genericError.description"),
-                level: AlertLevels.ERROR,
-                message: error?.message || t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.genericError.message")
-            }));
-        });
-    }
+        patchUserStore(id, [ patchData ])
+            .then(() => {
+                setEnabled(data.checked);
+
+                dispatch(addAlert<AlertInterface>({
+                    description: t("adminPortal:components.userstores.notifications." +
+                        "updateUserstore.success.description"),
+                    level: AlertLevels.SUCCESS,
+                    message: t("adminPortal:components.userstores.notifications." +
+                        "updateUserstore.success.message")
+                }));
+
+                // ATM, userstore operations run as an async task in the backend. Hence, The changes aren't 
+                // applied at once. As a temp solution, a notification informing the delay is shown here.
+                // TODO: Remove delay notification once backend is fixed.
```

Comment:
Can do the same as above.

Reply:
Referenced https://github.com/wso2/product-is/issues/9767 here.